src/client/src/components/homepage.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import HomePage from './homepage'
+import logo from '../logo.svg'
+import { shallow, configure, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16'
+
+configure({adapter: new Adapter()})
+
+describe('<HomePage />', () => {
+    const wrapper = shallow(<HomePage />)
```

Comment:
```
let suite = {}
describe('<HomePage />', () => {
    beforeEach(() => {
        suite.wrapper = shallow(<HomePage />)
    })
    afterEach(() => {
        // RESET ReactDOM completely
        suite = {}
    )
```

An environment of each test should be independent. It should be created every time from scratch before test, and cleared completely after it. This way you make sure, that tests won't influence each other. Think of them like scientific experiments, if some external behaviour will enter your tests, it's not trustworthy anymore.

Reply:
Done